packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -348,7 +348,7 @@ export interface AccountHistoryOptions {
   depth?: number
   no_rewards?: boolean
   token?: string
-  txtype?: string
+  txtype?: TxType | string
```

Comment:
i think just TxType because it should be restricted, we don't open for `string`

Reply:
It's fixed.